artifacts/fedora/fedora.go

```diff
@@ -195,6 +195,7 @@ const (
 	defaultInstancetype      = "u1.medium"
 	defaultPreferenceX86_64  = "fedora"
 	defaultPreferenceAarch64 = "fedora.arm64"
+	defaultPreferenceS390x   = "fedora.s390x"
```

Comment:
This doesn't exist yet in common-instancetypes, can you post a PR?

Reply:
Hi @lyarwood @0xFelix ,

I will move this commit to a new PR and continue the discussions there. I will keep this PR focused on the Ubuntu and CentOS artifacts only.

Since the build process for Fedora differs from that of Ubuntu and CentOS, could you please advise on how you would suggest adding preferences here for a new architecture? As far as I know, the common instance types also depend on the containerdisks repository during their testing. 

Looking forward to your reply. Thanks in advance!

Reply:
You are right that there is a chicken and egg problem. I would start by adding the containerdisks without environment variables for instancetypes/preferences. The second step would be to create suitable instancetypes and preferences and the last step would be to add them to the containerdisks once they are verified working.

Adding the containerdisks without environment variables as a first step should get you unblocked quickly.